src/util/LOG.cxx

```diff
@@ -13,7 +17,11 @@ LOG::~LOG()
 #ifndef DEBUG
     if (m_LogType != LOG_DEBUG)
 #endif
-      std::cout << Message << std::endl;
+#ifdef __ANDROID__
+      __android_log_print(ANDROID_LOG_INFO, "Cytopia", "%s", Message.str().c_str());
```

Comment:
Consider using small letter for local variable. (Message)

Reply:
done